packages/streaming-image-volume-loader/api-extractor.json

```diff
@@ -1,7 +1,7 @@
 {
   "extends": "../../api-extractor.json",
   "projectFolder": ".",
-  "mainEntryPointFilePath": "<projectFolder>/dist/cjs/index.d.ts",
+  "mainEntryPointFilePath": "<projectFolder>/dist/esm/index.d.ts",
```

Comment:
other `api-extractor`s use cjs, why this one is only esm, can we do it in another PR?

Reply:
Thought I had excluded that